bindings/node.js/src/kzg.cxx

```diff
@@ -152,7 +168,9 @@ Napi::Value LoadTrustedSetup(const Napi::CallbackInfo& info) {
 
   // Check that loading the trusted setup was successful
   if (ret != C_KZG_OK) {
-    Napi::Error::New(env, "Error loading trusted setup file: " + file_path).ThrowAsJavaScriptException();
+    std::ostringstream msg;
+    msg << "[C_KZG_RET::" << from_c_kzg_ret(ret) << "] Error loading trusted setup file: " << file_path;
```

Comment:
In reference to my other comment, this one is a little more complicated.

And now I see why you did it the way you did.

Maybe keep it simple and just print the return value, like this:
```suggestion
    msg << "Error loading trusted setup file: " << from_c_kzg_ret(ret);
```

Because above this, it will print the file path if it doesn't exist:

https://github.com/ethereum/c-kzg-4844/blob/e83ec0374ce95003603f3e552f15d10f1707408f/bindings/node.js/src/kzg.cxx#L142-L146